pkgs/applications/networking/p2p/peergos/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchurl, jre, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "peergos";
+  version = "0.14.1";
+
+  peergos = fetchurl {
+    url = "https://github.com/Peergos/web-ui/releases/download/v${version}/Peergos.jar";
+    hash = "sha256-oCsUuFxTAL0vAabGggGhZHaF40A5TLfkT15HYPiKHlU=";
+  };
+
+  dontUnpack = 1;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/share/java/
+    cp $peergos $out/share/java/peergos.jar
+
+    makeWrapper ${jre}/bin/java $out/bin/peergos --chdir $out/share/peergos/ --add-flags "-Djava.library.path=$out/share/peergos/native-lib/libtweetnacl.so -jar $out/share/java/peergos.jar"
+    mkdir -p $out/share/peergos/ && cd $out/share/peergos/
+    ${jre}/bin/java -jar $out/share/java/peergos.jar
```

Comment:
Is this supposed to be a test?

Reply:
should create `native-lib/libtweetnacl.so` 

anyway when running `$out/bin/peergos` it does not recognize this file and fallbacks to pure Java crypto library - see https://github.com/Peergos/Peergos/issues/849

Reply:
Please add a comment explaining that the file is generated.